src/index.html

```diff
@@ -73,5 +77,11 @@ <h2>What is the question?</h2>
         ></div>
       </section>
     </main>
+    <nav class="app-footer">
+      <span role="button" class="material-icons md-36">home</span>
+      <span role="button" class="material-icons md-36">bookmarks</span>
+      <span role="button" class="material-icons md-36">create</span>
+      <span role="button" class="material-icons md-36">face</span>
```

Comment:
I'd recommend using anchor elements since you want to link HTML pages.
```suggestion
      <a href="/" class="material-icons md-36">home</a>
      <a href="/..." class="material-icons md-36">bookmarks</a>
      <a href="/..." class="material-icons md-36">create</a>
      <a href="/..." class="material-icons md-36">face</a>
```